### DISCUSSION TITLE
keyboard shortcut - focus address bar

### DISCUSSION THREAD

**Discussion by Jayrgo** [Floorp-Projects/Floorp#788] [Feature Request]
it would be nice to have a keyboard shortcut to focus the address bar
e.g. vivaldi: 
![image](https://github.com/Floorp-Projects/Floorp/assets/13069032/6f474b3d-ecf4-4a8b-8dbc-a44f45c3b3f8)

**Comment by CutterKnife** [comment]
Thanks for the request.
If you have not disabled keyboard shortcuts in Firefox, you can use Ctrl(Cmd)+L to focus.

**Comment by Ryosuke-Asano** [comment]
I will add it
